creme/feature_extraction/agg.py

```diff
@@ -18,10 +19,15 @@ class Agg(base.Transformer):
     aggregate statistics in one go. The keys of the `dict` containing the aggregates is
     automatically guessed from `how`, `by`, and `on`.
 
+    ``shift``parameters add a defined lag to the input stream. The running statistics returns
```

Comment:
Only one backtick is needed now, so `shift` instead of ``shift``. This goes for the rest of the inline stuff. 

The comment should rather be: "The `shift` parameter determines the lag applied to the input stream.".